disperser/common/v2/blobstore/dynamo_store.go

```diff
@@ -0,0 +1,283 @@
+package blobstore
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	commondynamodb "github.com/Layr-Labs/eigenda/common/aws/dynamodb"
+	"github.com/Layr-Labs/eigenda/core"
+	"github.com/Layr-Labs/eigenda/disperser"
+	v2 "github.com/Layr-Labs/eigenda/disperser/common/v2"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/feature/dynamodb/attributevalue"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+)
+
+const (
+	StatusIndexName            = "StatusIndex"
+	OperatorDispersalIndexName = "OperatorDispersalIndex"
+	OperatorResponseIndexName  = "OperatorResponseIndex"
+
+	blobKeyPrefix     = "BlobKey#"
+	dispersalIDPrefix = "DispersalID#"
+	blobMetadataSK    = "BlobMetadata"
```

Comment:
So we're not using immutable design where blob metadata updates get a new unique sortable `SK` ie `BlobMetadata#UUID7` or `BlobMetadata#EpochSeconds`?

Reply:
yeah as discussed offline, some fields (BlobStatus) will be updated